test/test.js

```diff
@@ -1,14 +1,14 @@
 'use strict';
 var path = require('path');
-var readChunk = require('read-chunk');
 var test = require('ava');
 var isDoc = require('../');
-
+var read = require('fs').readFileSync;
 test(function (t) {
-	t.plan(2);
-
-	readChunk(path.join(__dirname, 'fixtures/test.doc'), 0, 8, function (err, buf) {
-		t.assert(!err, err);
-		t.assert(isDoc(buf));
-	});
-});
+	t.plan(3);
+    var buf=path.join(__dirname, 'fixtures/test.doc');
+    var buf2=path.join(__dirname, 'fixtures/test.xls');
+    var buf3=path.join(__dirname, 'fixtures/test.ppt');
+    t.assert(isDoc(read(buf)));
```

Comment:
Instead of `fs.readFileSync`, just use `fs.readFile` and make use of `async/await`.